demo/src/index.tsx

```diff
@@ -21,687 +21,134 @@ const renderFn = (expanded: boolean): JSX.Element => {
-                </AccordionItemPanel>
-            </AccordionItem>
-            <AccordionItem>
-                <AccordionItemHeading>
-                    <h3 className="u-position-relative">
-                        AccordionItemState
-                        <div className="accordion__arrow" role="presentation" />
-                    </h3>
-                </AccordionItemHeading>
-                <AccordionItemPanel>
-                    <table>
-                        <thead>
-                            <tr>
-                                <th>name</th>
-                                <th>type</th>
-                                <th>default</th>
-                                <th>description</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            <tr>
-                                <td>expanded</td>
-                                <td>Boolean</td>
-                                <td>false</td>
-                                <td>Expands this item on first render</td>
-                            </tr>
-                            <tr>
-                                <td>children</td>
-                                <td>Function</td>
-                                <td>null</td>
-                                <td>
-                                    Takes expanded state as argument for
-                                    conditional rendering
-                                </td>
-                            </tr>
-                        </tbody>
-                    </table>
-                </AccordionItemPanel>
-            </AccordionItem>
+        <h2 className="u-margin-top">Expanding multiple items at once</h2>
+
+        <p>
+            If you set <strong>allowMultipleExpanded</strong> to{' '}
+            <strong>true</strong> then the accordion will permit multiple items
+            to be expanded at once.
+        </p>
+
+        <Accordion allowMultipleExpanded={true}>
+            {placeholders.map((placeholder: Placeholder) => (
+                <AccordionItem key={placeholder.heading}>
+                    <AccordionItemHeading>
+                        <Arrow />
+                        {placeholder.heading}
+                    </AccordionItemHeading>
+                    <AccordionItemPanel>{placeholder.panel}</AccordionItemPanel>
+                </AccordionItem>
+            ))}
         </Accordion>
 
-        <h2 className="u-margin-top">Multi Accordion children</h2>
-
-        <Accordion>
-            <AccordionItem>
-                <AccordionItemHeading>
-                    <h3 className="u-position-relative">
-                        Components API
-                        <div className="accordion__arrow" role="presentation" />
-                    </h3>
-                </AccordionItemHeading>
-                <AccordionItemPanel>
-                    <Accordion allowMultipleExpanded={true}>
```

Comment:
I'm not sure, was it valuable to show that you can put accordions inside accordions?

I'd also make sure that we still have some demo content that has a link or two inside the panel so we can demonstrate that tabbing to those will still work.

Reply:
Yeah I would consider re-instating the nested one. My thinking was that this demo could afford to be a lot more minimal, and that by showing too many different patterns it was becoming overwhelming.

Reply:
Yes, good idea re simplifying.

Reply:
Links are a good idea... I'll whack some of those in there.

Reply:
Links there now @samanthaksanders. Going to merge now, and will revisit the nested accordiong when I come back to do the 'TODOs'